app/ui/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -259,6 +262,7 @@ public void onChanged(@Nullable List<Folder> folders) {
         initializeLayout();
         initializeFragments();
         displayViews();
+        channelUtils.updateChannels();
```

Comment:
This should probably not be done in the UI thread. K9 already needs too long to start.

Not sure if you received my first comment as I accidentally did a line note on a commit instead of a review on the PR.

Reply:
@ByteHamster 👍 Good point. Done.